canonical/rebac-admin
# WD-11350 - refactor: simplify creating panels

## Done

- insert portals into the app frame instead of an inner aside so we can simplify creating panels.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Go to groups or roles and click a "create ..." button
- the panel should appear as it did before.

## Details

https://warthogs.atlassian.net/browse/WD-11350
